src/odemis/driver/orsay.py

```diff
@@ -2799,24 +2760,32 @@ class Focus(model.Actuator):
     in focus distance in meter.
     """
 
-    def __init__(self, name, role, parent, rng, metadata, **kwargs):
+    def __init__(self, name, role, parent, rng, **kwargs):
         """
         Initialise Focus. Raises AttributeError exception if there is no _fib_beam sibling.
         :param (tuple (float, float)) rng: the range of the z axis
         """
         axes_def = {"z": model.Axis(unit="m", range=rng)}
         super().__init__(name, role, parent=parent, axes=axes_def, **kwargs)
 
-        self._metadata.update(metadata)  # store the metadata passed through the config file
-
         self.position = model.VigilantAttribute({"z": 0.0}, readonly=True, unit="m")
         self.parent._fib_beam.objectiveVoltage.subscribe(self._updatePosition)
 
         self.baseLensVoltage = 0.0  # V1, the objective lens voltage corresponding to a focus distance of 0
         # Changes during runtime
 
+        # Event that get's set when the position VA changes value while performing a move using _doMoveAbs
+        self._position_changed = threading.Event()
+        self._position_changed.clear()
+
         self._executor = CancellableThreadPoolExecutor(max_workers=1)
 
+        self.updateMetadata({})
+
+    def updateMetadata(self, md):
+        md[model.MD_CALIB] = 0.18e6
```

Comment:
This value should be in `md`. If not, too bad. You typically don't need to override `updateMetadata()`. If you want to be extra careful, you could override it to check that the new MD_CALIB is valid (eg, it's a float).